subworkflows/local/utils_nfcore_atacseq_pipeline/main.nf

```diff
@@ -147,33 +148,6 @@ def validateInputParameters() {
     }
 }
 
-//
-// Validate channels from input samplesheet
-//
-def validateInputSamplesheet(input) {
```

Comment:
Is this function not used?

Reply:
No. RNAseq, which we use as reference isn't using it either.